.travis.yml

```diff
@@ -1,8 +1,10 @@
-language: generic
-
+language: python
+python:
+    - 3.6
 sudo: required
 services:
     - docker
-
+install:
+    - pip install docker pytest requests
```

Comment:
should this be moved in to the `make test` command? Otherwise it seems to make it slightly more challenging to run the tests